ngraph/python/src/pyngraph/node_factory.cpp

```diff
@@ -69,7 +69,7 @@ namespace
 
```

Comment:
@postrational @tomdol This method has next arguments:
```
std::shared_ptr<ngraph::Node> create(const std::string op_type_name,
                                             const ngraph::NodeVector& arguments,
                                             const py::dict& attributes = py::dict())
```
Should we use `OutputVector` instead of `NodeVector`? `NodeVector` doesn't work for operations with several outputs.

Reply:
This might be impossible without a solid refactor of the python api. This method is exposed to python and each operator is created by calling it. It operates on a list of nodes https://github.com/openvinotoolkit/openvino/blob/cc23e6043a632c3dded3732e131bc0fac22ae05a/ngraph/python/src/ngraph/utils/node_factory.py#L22 
Also in the ops.py file the operator nodes are created using NodeInput type https://github.com/openvinotoolkit/openvino/blob/185cafb4cc9ab7ed779dedf8b9f653aede0e9301/ngraph/python/src/ngraph/opset1/ops.py#L62 which is either a Node or a number.
Not sure if this is doable as an extension of this PR.

Reply:
Yes I agree that it should not be a part of this PR. But maybe we should do it. What do you think?

Reply:
I guess that's something to consider seriously. Multiout operators are important and perhaps we don't have a test that covers them.